fetch_umbrella_build_and_test.sh

```diff
@@ -27,4 +27,14 @@ cmake .. -DGUI=0 -DCMAKE_BUILD_TYPE=$TRAVIS_BUILD_TYPE $OPTIONS
 make lllc solc soljson soltest
 
 
-./solidity/test/soltest
+# Test runs disabled for macos for now,
+# we need to find a way to install eth.
```

Comment:
Wouldn't that just be `brew install cpp-ethereum`?
Or we could just start with a hardcoded unzip of https://github.com/ethereum/webthree-umbrella/releases/download/v1.2.9/cpp-ethereum-osx-yosemite-v1.2.9.zip